### ISSUE TITLE
I trained the network myself with sbd dataset and it can not detect any object

### ISSUE THREAD

**Issue by beeying** [zju3dv/snake#209]
When I used pre-trained network it's work. but when I trained the network from scratch with the default setting, it didn't work.
How can I do to get a better result?
I did with the sbd dataset.

**Comment by pengsida** [comment]
How many epochs do you train the network?
Could you visualize the loss curves?
